client/src/pages/Login/LoginPage.jsx

```diff
@@ -0,0 +1,76 @@
+import { React, useState } from "react";
+import "../Landing/Landing.css";
+import "../../index.css";
+import { useNavigate } from "react-router-dom";
+
+const LoginPage = () => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const navigate = useNavigate();
+
+  const handleInputChange = (fieldName) => (e) => {
+    if (fieldName === "username") {
+      setUsername(e.target.value);
+    } else if (fieldName === "password") {
+      setPassword(e.target.value);
+    }
+  };
+
+  const handleLogin = async () => {
+    const baseApiUrl = process.env.REACT_APP_API_URL;
+    try {
+      const response = await fetch(`${baseApiUrl}/login`, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ username, password }),
+      });
+
+      if (!response.ok) {
+        throw new Error("Login failed");
+      }
+
+      const data = await response.json();
+      const { token } = data;
+      sessionStorage.setItem("token", token);
+
+      navigate("/dashboard");
+    } catch (error) {
+      // TODO handle login error
+      console.error("Login error: ", error);
+    }
+  };
+
+  return (
+    <div className="landing-body">
+      <div className="background-blue item-container parent-center">
+        <form>
+          <label>
+            Username:
+            <input
+              type="text"
+              value={username}
+              onChange={handleInputChange("username")}
+            />
+          </label>
+          <br />
+          <label>
+            Password:
+            <input
+              type="password"
+              value={password}
+              onChange={handleInputChange("password")}
```

Comment:
is this hard-coded for the time being?

Reply:
@justfrankie no, the password value changes with input and it sets the new password value as the input value 
```
  const [password, setPassword] = useState("");
  const navigate = useNavigate();

  const handleInputChange = (fieldName) => (e) => {
    if (fieldName === "username") {
      setUsername(e.target.value);
    } else if (fieldName === "password") {
      setPassword(e.target.value);
    }
  }; 
``` 
[here](https://github.com/hsu-isaac/fitconnect/pull/15/files#diff-b6cc423015dbe4e6e8025044c9a0b8c3557414ab7b680729f92ce37041d4dbc0R9-R18)